stubs/untangle/untangle.pyi

```diff
@@ -0,0 +1,37 @@
+from _typeshed import Incomplete
+from collections.abc import Iterator
+from typing import Any
+from xml.sax import handler
+
+def is_string(x: Any) -> bool: ...
```

Comment:
When it's safe to pass literally any object into a function, we should use `object` rather than `Any`. `Any` should generally be reserved for situations where something is inexpressable in the type system.

```suggestion
def is_string(x: object) -> bool: ...
```